firedrake/mesh.py

```diff
@@ -434,9 +722,8 @@ def add_overlap():
             nfacets = plex.getConeSize(cStart)
 
         # TODO: this needs to be updated for mixed-cell meshes.
-        nfacets = self.comm.allreduce(nfacets, op=MPI.MAX)
+        nfacets = self._comm.allreduce(nfacets, op=MPI.MAX)
```

Comment:
All these should use the public `self.comm`.

Reply:
changed.